content/webapp/services/prismic/transformers/json-ld.ts

```diff
@@ -135,6 +138,42 @@ export function eventLd(event: Event): JsonLdObj[] {
     });
 }
 
+export function eventLdContentApi(event: EventContentApi): JsonLdObj[] {
```

Comment:
This version contains less than the Prismic API fed one - I've removed "cost", "contributors" and "caption", as those fields weren't pulled in from our API already, and I don't think they are worth the extra data being requested on every call for a JSON-LD reason.